ksql-functional-tests/src/test/resources/query-validation-tests/sum.json

```diff
@@ -201,14 +201,14 @@
       "inputs": [
         {"topic": "test_topic", "key": 0, "value": {"value":[1.0, 1.0]}},
         {"topic": "test_topic", "key": 0, "value": {"value":[2.0, 2.0]}},
-        {"topic": "test_topic", "key": 0, "value": {"value":[9223372036854775807.0, 1.0]}},
+        {"topic": "test_topic", "key": 0, "value": {"value":[922337203685.0, 1.0]}},
```

Comment:
this test previously was incorrect - JSON serialized it as a long, which can fit the old value, but since it's specified a as a double this number actually has higher precision than a double can hold